.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+## Description
+
+Please include a summary of the change, motivation and context
+
+- **on a feature**: describe the feature and how this change fits in it, e.g. this PR makes kafka message.max.bytes configurable to better support batching
+- **on a refactor**: describe why this is better than previous situation e.g. this PR changes logic for retry on healthchecks to avoid false positives
+- **on a bugfix**: link relevant information about the bug (github issue or slack thread) and how this change solves it e.g. this change fixes #99999 by adding a lock on read/write to avoid data races.
+
+### Testing
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
```

Comment:
> Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

We don't want to be reproducing anything in tests. In this sections users should summarize what has been tested  and maybe what should be tested more. We should rely on the CI to test the changes and not on users telling us it works - this should be rather exceptional.

Reply:
Agree. I think this section can be misleading as even if the user manages to test this manually we don't want to to go though that process.